demos/next-11/.eslintrc

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "next",
+  "root": true
```

Comment:
In case anyone else is interested in what this property does (because I didn't know offhand either), from the ESLint docs site:

> By default, ESLint looks for configuration files in all parent folders up to the root directory. This can be useful if you want all of your projects to follow a certain convention, but can sometimes lead to unexpected results. To limit ESLint to a specific project, place "root": true inside the .eslintrc.* file or eslintConfig field of the package.json file or in the .eslintrc.* file at your project’s root level. ESLint stops looking in parent folders once it finds a configuration with "root": true.

@orinokai This is a bit of a nitpick, but it might be worth adding a small note here about ESLint not looking for configuration in parent directories past this point with this setting since this info is a little buried in the docs